package spob;

message ConstructTree {
  required uint32 max_rank = 1;
  required uint64 count = 2;
  repeated uint32 ancestors = 3;
}